docs/content/en/blog/releases/v0.43.0.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Release v0.43.0"
+linkTitle: "Release v0.43.0"
+date: 2023-03-27
+description: >
+ Release v0.43.0
+---
+
+## Changes since v0.42.1
+
+### New Features
+
+* Enable to filter application by its piped ([#4244](https://github.com/pipe-cd/pipecd/pull/4244))
+* Enable to ignore manifests' fields from kubernetes' drift detection ([#4249](https://github.com/pipe-cd/pipecd/pull/4249))
+
+You're now can configure the piped to ignore manifests' fields from drift detection feature via config in that application configuration (the `app.pipecd.yaml` file).
+
+```yaml
+spec:
+  ...
+  driftDetection:
+    ignoreFields:
+      - spec.replicas
```

Comment:
Oh, sorry, we should have fixed this.

Reply:
plz https://github.com/pipe-cd/pipecd/pull/4298